.gitignore

```diff
@@ -1 +1,2 @@
 /data
+custom-backend.env
```

Comment:
This is included where exactly?

Reply:
I added additional variables to my `backend.env`, so I rename the file to `custom-backend`
We can use the `backend.example.env` approach instead